openshift-start-configured-cluster

```diff
@@ -7,10 +7,14 @@ source $DIR/config
 
 OC_CONFIG=`mktemp -d --suffix=.openshift-config`
 
-cd $GOPATH/src/github.com/openshift/origin
-export PATH="$( source hack/lib/init.sh; echo "${OS_OUTPUT_BINPATH}/$( os::build::host_platform )/" ):${PATH}"
+if [ ! -f oc ]; then
+   wget -N -L https://github.com/openshift/origin/releases/download/v3.10.0-rc.0/openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit.tar.gz
+   tar -zxf openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit.tar.gz
+   mv openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit/oc ./
+   rm -rf openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit.tar.gz
+fi
 
-oc cluster up --base-dir=$OC_CONFIG --write-config=true --tag=latest --public-hostname $OC_PUBLIC_HOSTNAME --server-loglevel=10
+./oc cluster up --base-dir=$OC_CONFIG --write-config=true --tag=latest --public-hostname $OC_PUBLIC_HOSTNAME --server-loglevel=10
```

Comment:
you do not need -tag latest if you pull the latest image, also latest now causes v3.11 development branch to be installed (which Is ok imho)